testapi.pm

```diff
@@ -529,23 +530,29 @@ sub assert_screen_change(&@) {
 
 =for stopwords stilltime
 
-  wait_still_screen([$stilltime_sec [, $timeout [, $similarity_level]]]);
+  wait_still_screen([$stilltime | [stilltime => $stilltime]] [, $timeout] | [timeout => $timeout]] [, similarity_level => $similarity_level] [, no_wait => $no_wait]);
 
 Wait until the screen stops changing.
 
-Returns true if screen is not changed for given $stilltime (in seconds) or undef on timeout.
+See C<assert_screen> for C<$no_wait>.
+
+Returns true if screen is not changed for given C<$stilltime> (in seconds) or undef on timeout.
 Default timeout is 30s, default stilltime is 7s.
 
 =cut
 
 sub wait_still_screen {
-    my $stilltime        = shift || 7;
-    my $timeout          = shift || 30;
-    my $similarity_level = shift || (get_var('HW') ? 44 : 47);
-
-    bmwqemu::log_call(stilltime => $stilltime, timeout => $timeout, simlvl => $similarity_level);
-
-    $timeout = bmwqemu::scale_timeout($timeout);
+    my $stilltime = looks_like_number($_[0]) ? shift : undef;
```

Comment:
I think it's safe to put `7` here as well, so you don't have to check if it's `undef` later

Reply:
done